crates/server/src/bin/warg-server.rs

```diff
@@ -24,40 +24,36 @@ struct Args {
     #[arg(short, long, env = "LISTEN", default_value = "127.0.0.1:8090")]
     listen: SocketAddr,
 
-    /// Enable content service, with storage in the given directory
+    /// The content storage directory to use.
     #[arg(long, env = "CONTENT_DIR")]
-    content_dir: Option<PathBuf>,
+    content_dir: PathBuf,
 
     /// The data store to use for the server.
     #[arg(long, env = "DATA_STORE", default_value = "memory")]
     data_store: DataStoreKind,
 
     /// The database connection URL if data-store is set to postgres.
     ///
-    /// Prefer using database-url-file, or environment variable variation,
+    /// Prefer using `database-url-file`, or environment variable variation,
     /// to avoid exposing sensitive information.
     #[cfg(feature = "postgres")]
     #[arg(long, env = "DATABASE_URL")]
     database_url: Option<String>,
 
     /// The path to the operator key.
-    ///
-    /// Takes precedence over database-url.
     #[cfg(feature = "postgres")]
-    #[arg(long, env = "DATABASE_URL_FILE")]
+    #[arg(long, env = "DATABASE_URL_FILE", conflicts_with = "database_url")]
     database_url_file: Option<PathBuf>,
 
     /// The operator key.
     ///
-    /// Prefer using warg-operator-key-file, or environment variable variation.
-    #[arg(long, env = "WARG_OPERATOR_KEY")]
-    warg_operator_key: Option<String>,
+    /// Prefer using `operator-key-file`, or environment variable variation.
+    #[arg(long, env = "OPERATOR_KEY")]
+    operator_key: Option<String>,
```

Comment:
Try to make this `Option<PrivateKey>` so that we never treat the secret as a string in Rust code and e.g. debug print it.

Reply:
Unfortunately clap requires `Clone` on option fields, which `PrivateKey` doesn't implement intentionally.

Going to leave as-is now with the hope that this option will be removed in the near-term in favor of pulling the key from a key ring.